src/main/java/core/basesyntax/strategy/handler/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationStrategy;
+
+public class BalanceOperation implements OperationStrategy {
+    @Override
+    public void makeFruitTransactionOperation(FruitTransaction fruitTransaction) {
+        Storage.STORAGE.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());

```

Comment:
Made same direct access to storage, and had reject from mentors) Had to implement dao package. But it's up to you)

Reply:
no problem at all) rejects are my path to success))